docs/_data/config-prague-2019.yaml

```diff
@@ -0,0 +1,108 @@
+name: Prague
+
+shortcode: prague # prague australia?
+templatecode: eu # HACK
+year: 2019
+city: Prague
+local_area: Czech Republic
+area: Europe
+area_adj: European
+tz: CEST
+email: prague@writethedocs.org
+color: coral
+
+photos:
+  default: _static/2019/assets/headers/prague-group.png
+
+tickets:
+  community:
+    price: €500
+  corporate:
+    price: €250
+  independent:
+    price: €150
+  student:
+    price: €75
+
+sponsorship:
+  first_draft:
+    price: €500
+  second_draft:
+    price: €1000
+  publisher:
+    price: €3000
+  patron:
+    price: €6000
+  keystone:
+    price: €12000
+  job_fair:
+    price: €800
+    half: €400
+  lanyard:
+    price: TBD
+  writing_day:
+    price: TBD
+
+date: # how do we handle these? human readable would be nice.
```

Comment:
I *think* some of these are not quite needed, prob `day_four` as we hack that into a multi day day three :-p

Reply:
Removed it, seems fine.